comparison.md

```diff
@@ -109,19 +106,19 @@ AssertJ provides assertions for several libraries' types. As of this writing,
 Truth includes assertions for [Guava] and [Protocol Buffers].
 
 Both have third-party extensions, such as for Android types
-([AssertJ][AssertJ-Android], [Truth][Truth-Android]). I don't have a feel for
-the overall size of each ecosystem.
+([AssertJ][AssertJ-Android], [Truth][Truth-Android]).
+It's unclear how large each ecosystem is overall.
 
 ### Platform support (Android, GWT)
 
 AssertJ supports Android (though I had to use 2.x because the dexer rejected
```

Comment:
This is still in first person, but it seems like unrelated details by now? @cpovirk?

Reply:
The right thing for me to do here is to try 3.0 with some other build system and see if it works, then file a bug against AssertJ about the problems. In the meantime, I want to point out that AssertJ might not actually work with Android in practice, despite its best attempts, but that I'm not sure how widespread the problem is.